pages/windows/slmgr.md

```diff
@@ -0,0 +1,7 @@
+# slmgr
+
+> This command is an alias of `slmgr.vbs`.
```

Comment:
```suggestion
> This command is an alias of `slmgr.vbs`.
> More information: <https://docs.microsoft.com/windows-server/get-started/activation-slmgr-vbs-options>.
```

Reply:
Is there any reason to put more information things into this alias page?

Reply:
There are 2 reasons:

- current style guide [requires](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#aliases) this alias page template (but it can be modified)
- it's faster to open manual when it's in alias page